internal/services/datafactory/data_factory_integration_runtime_self_hosted_resource.go

```diff
@@ -90,14 +74,12 @@ func resourceDataFactoryIntegrationRuntimeSelfHosted() *pluginsdk.Resource {
 				},
 			},
 
-			// todo: rename to authorization_key_primary in v3.0
-			"auth_key_1": {
+			"authorization_key_primary": {
 				Type:     pluginsdk.TypeString,
 				Computed: true,
 			},
 
-			// todo: rename to authorization_key_secondary in v3.0
-			"auth_key_2": {
+			"authorization_key_secondary": {
```

Comment:
Would these be better renamed as `primary_authorization_key` and `secondary_authorization_key`?